tests/cli/ui/data/list_wf_runs.txt

```diff
@@ -1,6 +1,6 @@
-                                                                                
-  Workflow Run ID   Status          Succeeded Tasks   Start Time                
- ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
-  wf1               mocked status   x/y               Fri Feb 24 08:26:07 2023  
-  wf2               mocked status   x/y                                         
-                                                                                
+                                                                                   
+  Workflow Run ID   Status          Succeeded Tasks   Start Time                   
+ ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
+  wf1               mocked status   x/y               Fri Feb 24 08:26:07 2023     
+  wf2               mocked status   x/y                                            
+                                                                                   
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Is this whitespace change related to my previous question?

Reply:
Aah yeah, good catch that's exactly what's happening! Empty column removed in [e552428](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/320/commits/e552428122a02399f6d87cae710bd93b436689db)